rbs/prism/MethodTypeToParserNodePrism.h

```diff
@@ -14,42 +14,40 @@ extern "C" {
 namespace sorbet::rbs {
 
 class MethodTypeToParserNodePrism {
-  core::MutableContext ctx;
-  Parser parser;
-  const parser::Prism::Parser *prismParser; // For Prism node creation
+    core::MutableContext ctx;
+    Parser parser;
+    const parser::Prism::Parser *prismParser; // For Prism node creation
```

Comment:
Ref would be preferable

```suggestion
    const parser::Prism::Parser &prismParser; // For Prism node creation
```